### PR TITLE
feat: add process-cpp

### PR DESCRIPTION
**Pull request by cat-master21**

### CODE REVIEW

File: anda/lib/process-cpp/process-cpp.spec [hunk 1]
```diff
@@ -0,0 +1,84 @@
+%global forgeurl https://gitlab.com/ubports/development/core/lib-cpp/process-cpp
+%global commit ee6d99a3278343f5fdcec7ed3dad38763e257310
+%forgemeta
+
+Name:          process-cpp
+Version:       3.0.2
```
Comment (windowsboy111) [inline anda/lib/process-cpp/process-cpp.spec:6]: `3.0.2` is unreleased. I'm not sure if we should do it like this… Is is a must to use the latest commit?

**Review by windowsboy111 — COMMENTED** [review]
Auto update script:
```ts
let o = get("https://gitlab.com/api/v4/projects/16508826/repository/tags").json_arr()[0];
rpm.global("commit", o.commit.id);
rpm.version(o.name);
```

File: anda/lib/process-cpp/process-cpp.spec (hunk 1 above)
Comment (cat-master21) [reply anda/lib/process-cpp/process-cpp.spec:6]: My bad it looked like `3.0.2` was released https://gitlab.com/ubports/development/core/lib-cpp/process-cpp/-/blob/main/debian/changelog

Comment (windowsboy111) [reply anda/lib/process-cpp/process-cpp.spec:6]: I mean it does say
```
process-cpp (3.0.2) UNRELEASED; urgency=medium
```
unreleased in caps right there… so…

File: anda/lib/process-cpp/process-cpp.spec [hunk 2]
```diff
@@ -0,0 +1,84 @@
+%global forgeurl https://gitlab.com/ubports/development/core/lib-cpp/process-cpp
+%global commit ee6d99a3278343f5fdcec7ed3dad38763e257310
```
Comment (windowsboy111) [inline anda/lib/process-cpp/process-cpp.spec:2]: I think same issue here unfortunately, wrong commit

Comment (cat-master21) [reply anda/lib/process-cpp/process-cpp.spec:2]: Meaning what? There are some changes/commits not in `3.0.1`.

Comment (windowsboy111) [reply anda/lib/process-cpp/process-cpp.spec:2]: Hmm, maybe just name the commit `3.0.2^%commit`

**Review by windowsboy111 — APPROVED** [review]

**Pull request merged**